front/src/modules/simulationResult/components/SpeedSpaceChart/SpeedSpaceChart.tsx

```diff
@@ -179,18 +169,7 @@ export default function SpeedSpaceChart({
    */
   useEffect(() => {
     if (resetChart) {
-      if (rotate) {
-        // cancel rotation and redraw the train
-        toggleRotation();
-      } else if (chart && setSharedXScaleDomain) {
-        setSharedXScaleDomain((prevState) => ({
-          ...prevState,
-          current: prevState.initial,
-          source: undefined,
-        }));
-      } else {
-        createChartAndTrain();
-      }
+      createChartAndTrain();
```

Comment:
Be careful, you need to keep this, for the case you have zoomed in or out before clicking on reset ;)
```suggestion
       if (chart && setSharedXScaleDomain) {
        setSharedXScaleDomain((prevState) => ({
          ...prevState,
          current: prevState.initial,
          source: undefined,
        }));
      } else {
        createChartAndTrain();
      }
```